src/goto-checker/symex_bmc_incremental_one_loop.h

```diff
@@ -10,6 +10,7 @@
 #define CPROVER_GOTO_CHECKER_SYMEX_BMC_INCREMENTAL_ONE_LOOP_H
 
 #include "symex_bmc.h"
+#include <util/ui_message.h>
```

Comment:
Guess you don't need this one anymore.

Reply:
Needed for `ui_message_handlert::uit` declaration